components/edit-collective/sections/SetupTwoFactorAuth.js

```diff
@@ -0,0 +1,242 @@
+                      defaultMessage="Step one: scan this QR code with an authenticator app such as Google Authenticator"
+                    />
+                  </H3>
+                  <QRCode value={otpauth_url} renderAs="svg" size="256" level="L" includeMargin />
+                </Box>
+                <Box>
+                  <H3>
+                    <FormattedMessage
+                      id="TwoFactorAuth.Setup.StepTwo"
+                      defaultMessage="Step two: enter the code from your authentication app"
+                    />
+                  </H3>
+                  <Container>
+                    <Formik validate={validate} initialValues={initialValues} onSubmit={submit}>
+                      {formik => {
+                        const { values, handleSubmit, errors, touched } = formik;
+
+                        return (
+                          <Form>
+                            <StyledInputField
+                              name="TOTPCode"
+                              htmlFor="TOTPCode"
+                              error={touched.TOTPCode && errors.TOTPCode}
+                              label={intl.formatMessage(messages.inputLabel)}
+                              value={values.TOTPCode}
+                              required
+                              mt={4}
+                              mb={3}
+                            >
+                              {inputProps => (
+                                <Field
+                                  as={StyledInput}
+                                  {...inputProps}
+                                  minWidth={300}
+                                  minHeight={100}
+                                  fontSize={'H1'}
+                                  noNumberArrows
+                                  placeholder="123456"
+                                />
+                              )}
+                            </StyledInputField>
+
+                            <Flex justifyContent={['center', 'left']} mb={4}>
+                              <StyledButton
+                                fontSize="13px"
+                                minWidth="148px"
+                                minHeight="36px"
+                                buttonStyle="primary"
+                                type="submit"
+                                onSubmit={handleSubmit}
+                              >
+                                <FormattedMessage id="TwoFactorAuth.Setup.Form.VerifyButton" defaultMessage="Verify" />
+                              </StyledButton>
+                            </Flex>
+                          </Form>
+                        );
+                      }}
+                    </Formik>
+                  </Container>
+                </Box>
+              </Container>
+            </Fragment>
+          )}
+        </Flex>
+      </Flex>
+    );
+  }
+}
+
+const addTwoFactorAuthToAccountMutation = graphql(
+  gqlV2`
```

Comment:
For prettier to do its job

```suggestion
  gqlV2/* GraphQL */ `
```